src/models/IN_dataGenerator_random.py

```diff
@@ -0,0 +1,326 @@
+from __future__ import print_function
```

Comment:
what's the difference between this script and the `train_model.py` script? Can we merge the two into one?

Reply:
Thanks for reviewing and suggestions.

The `src/models/IN_dataGenerator_random.py` aimed to randomly split the train test data without the limitations of h5 file constraints. Because in the previous splitting we always select several files as test and the rest as training, and the data inside one h5 file will always be counted as test or train. The random split will first store all the data together, and shuffle and split them into train and test (this is what `src/utils/generate_train_test.py` does), then `IN_dataGenerator_random.py` will load those split data for training. In this way data inside one h5 file can also be split into train and test.

I can merge `IN_dataGenerator_random.py` to train_model.py and set a flag to decide whether we need to do random split(split happens within h5 file) or normal split(split happens outside h5 file). Also rename the `src/utils/generate_train_test.py` so that it can show more clear about it's function